integration-tests/mongodb-panache-kotlin/src/test/kotlin/io/quarkus/it/mongodb/panache/MongodbPanacheResourceTest.kt

```diff
@@ -4,27 +4,24 @@ import com.fasterxml.jackson.databind.ObjectMapper
 import com.fasterxml.jackson.databind.SerializationFeature
 import com.fasterxml.jackson.datatype.jdk8.Jdk8Module
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
-import io.quarkus.it.mongodb.panache.book.BookDetail
-import io.quarkus.it.mongodb.panache.person.Person
-import io.quarkus.test.common.QuarkusTestResource
-import io.quarkus.test.junit.QuarkusTest
-import io.quarkus.test.mongodb.MongoTestResource
 import io.restassured.RestAssured
 import io.restassured.RestAssured.get
 import io.restassured.common.mapper.TypeRef
 import io.restassured.config.ObjectMapperConfig
 import io.restassured.parsing.Parser
 import io.restassured.response.Response
-import java.util.Calendar
-import java.util.Collections
-import java.util.Date
-import java.util.GregorianCalendar
+import java.util.*
```

Comment:
Let's try and avoid these please. I know the IDE did it automatically, but we try hard to avoid star imports. I have changed the IDE's config to only add them after 50 classes under the same package.

Reply:
I didn't even catch this! I probably need to tweak the kotlin settings in IntelliJ. I did the Java ones as mentioned in the contributing guide. I'll fix this when I get to my desk this morning.

Reply:
@geoand does the `impsort` goal address this? I had closed the IDE completely and ran `mvn import:sort` on the entire codebase before I committed anything. Or does that goal only work on Java source files?

Reply:
Unfortunately it does not :(